hazelcast-documentation/src/HazelcastSimulator/ExecutingASimulatorTest.md

```diff
@@ -1,185 +1,310 @@
-
-        operationSelectorBuilder.addOperation(Operation.PUT, putProb).addDefaultOperation(Operation.GET);
-    }
-
-    @Warmup
-    public void warmup() {
-        log.info("======== WARMUP =========");
-        log.info("Map size is:" + map.size());
-    }
-
-    @Verify
-    public void verify() {
-        log.info("======== VERIFYING =========");
-        log.info("Map size is:" + map.size());
-
-        for (int i = 0; i < maxKeys; i++) {
-            assertEquals(map.get(i), "value" + i);
-        }
-    }
-
-    @Teardown
-    public void teardown() throws Exception {
-        log.info("======== TEAR DOWN =========");
-        map.destroy();
-        log.info("======== THE END =========");
-    }
-
-    @RunWithWorker
-    public AbstractWorker<Operation> createWorker() {
-        return new Worker();
-    }
-
-    private class Worker extends AbstractWorker<Operation> {
-
-        public Worker() {
-            super(operationSelectorBuilder);
-        }
-
-        @Override
-        protected void timeStep(Operation operation) {
-            int key = randomInt(maxKeys);
-            switch (operation) {
-                case PUT:
-                    map.put(key, "value" + key);
-                    break;
-                case GET:
-                    map.get(key);
-                    break;
-                default:
-                    throw new UnsupportedOperationException("Unknown operation" + operation);
-            }
-        }
-
-    }
-
-    public static void main(String[] args) throws Throwable {
-        ExampleTest test = new ExampleTest();
-        new TestRunner<ExampleTest>(test).run();
-    }
-}
+class=com.hazelcast.simulator.tests.map.StringStringMapTest
+threadCount=10
+keyLocality=Random
+keyLength=300
+valueLength=300
+keyCount=100000
+putProb=0.2
+basename=map
 ```
 
+* The property `class` defines the actual test case and the rest are the properties you want to bind in your test. If a
+property is not defined in this file, the default value of the property given in your test code is used. Please see the `properties` comment in the StringStringMapTest. You will see folllowings.
 
-### Editing the simulator.properties File
-
+```
+    // properties
+    public int keyLength = 10;
+    public int valueLength = 10;
+    public int keyCount = 10000;
+    public int valueCount = 10000;
+    public String basename = "stringStringMap";
+    public KeyLocality keyLocality = KeyLocality.RANDOM;
+    public int minNumberOfMembers = 0;
+```
  
-In the case of Amazon EC2, you need to consider the following properties.
+After you created test.properties and filled with your properties succesfully, you need configure your simulator via simulator.properties.
+
+Run this command and create file in the working directory.
+
+	cat > simulator.properties
+	
+Copy these lines into property file and fill with your properties.
 
 ```
-CLOUD_IDENTITY=~/ec2.identity
-CLOUD_CREDENTIAL=~/ec2.credential
+CLOUD_PROVIDER=aws-ec2
```

Comment:
It is not acceptable to 'fill in with your properties'

Please give working defaults. Now the user is completely stuck because he needs to figure out what he wants.